dashboard/src/components/AppView/AccessURLTable/AccessURLTable.test.tsx

```diff
@@ -178,6 +178,41 @@ context("when the app contains ingresses", () => {
     expect(wrapper.find("a").findWhere(a => a.prop("href") === "http://foo.bar/ready")).toExist();
     expect(wrapper).toMatchSnapshot();
   });
+
+  it("should show the table with available ingresses without anchors if a regex is present in the path", () => {
+    const ingress = {
+      kind: "Ingress",
+      metadata: {
+        name: "foo",
+        selfLink: "/ingresses/foo",
+      },
+      spec: {
+        rules: [
+          {
+            host: "foo.bar",
+            http: {
+              paths: [{ path: "/ready(/|$)(.*)" }],
+            },
+          },
+        ],
+      } as IIngressSpec,
+    } as IResource;
+    const ingressItem = { isFetching: false, item: ingress };
+    const url = ingress.metadata.selfLink;
+    const ingressRefs = [{ name: "svc", getResourceURL: jest.fn(() => url) } as any];
+    const state = { kube: { items: { [url]: ingressItem } } };
+    const store = getStore(state);
+    const wrapper = mountWrapper(
+      store,
+      <AccessURLTable {...defaultProps} ingressRefs={ingressRefs} />,
+    );
+    expect(wrapper.find("Table")).toExist();
+    expect(
+      wrapper.find("a").findWhere(a => a.prop("href") === "http://foo.bar/ready(/|$)(.*)"),
+    ).not.toExist();
+    expect(wrapper.find("span").first()).toHaveText("http://foo.bar/ready(/|$)(.*)");
```

Comment:
This is a bit risky, it would fail if we add any other `span` before. Better to do `wrapper.find("span").findWhere(s => s.text() === "http://...` so we find it wherever it is.

Reply:
The span element seems not to have a .text(), so I first have to render the cheerio component and then access the actual text: `wrapper.find("span").findWhere(s => s.render().text() ...`, if you have a better solution, please let me know.

Reply:
I double checked and the method `.text()` should be present (in fact, it should exist for any element). Tried this and worked:

```
    const matchingSpans = wrapper.find("span").findWhere(s => s.text().includes("foo.bar/ready"));
```

Reply:
Mmm, don't know either... but now it's working as you mention. Perhaps a misconfiguration in my linter led me to think there was an error with the .text() method. Thank you for your comments!